lib/benchee/formatters/console.ex

```diff
@@ -46,23 +46,25 @@ defmodule Benchee.Formatters.Console do
   iex> suite = %Benchee.Suite{
   ...>   scenarios: scenarios,
   ...>   configuration: %Benchee.Configuration{
-  ...>     formatter_options: %{
-  ...>       console: %{comparison: false, extended_statistics: false}
-  ...>     },
   ...>     unit_scaling: :best,
   ...>   }
   ...> }
-  iex> Benchee.Formatters.Console.format(suite)
+  iex> Benchee.Formatters.Console.format(suite, %{comparison: false, extended_statistics: false})
   [["\n##### With input My input #####", "\nName             ips        average  deviation         median         99th %\n",
   "My Job           5 K         200 ns    ±10.00%         190 ns      300.10 ns\n",
   "Job 2         2.50 K         400 ns    ±20.00%         390 ns      500.10 ns\n"]]
 
   ```
 
   """
-  @spec format(Suite.t()) :: [any]
-  def format(%Suite{scenarios: scenarios, configuration: config}) do
-    config = console_configuration(config)
+  @spec format(Suite.t(), map) :: [any]
```

Comment:
We might want to consider using `@impl Benchee.Formatter` here (and for `write/2`) so we get the compiler check that we're implementing the behaviour correctly.